docs/dev/stream/operators/windows.md

```diff
@@ -797,7 +797,7 @@ DataStream<SensorReading> input = ...;
 
 input
   .keyBy(<key selector>)
-  .timeWindow(<window assigner>)
+  .timeWindow(<time size>)
```

Comment:
How about using "duration" instead of "time size". I think "time size" is not a commonly used term...

Reply:
accept, hold on...